.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: python
+python:
+  - "2.7"
+env:
+  - DJANGO=1.4.20
+  - DJANGO=1.5.12
+  - DJANGO=1.6.11
+  - DJANGO=1.7.8
+  - DJANGO=1.8.2
+install:
+  - pip install -q Django==$DJANGO --use-mirrors
+  - pip install pep8 --use-mirrors
+  - pip install pyflakes
+  - pip install -q -e . --use-mirrors
+#before_script:
+  #- "pep8 --exclude=migrations --ignore=E501,E225 datatableview"
```

Comment:
Not willing to pep8-ify or pyflakes the source tree, but this is here for possible reference.